guvnor-ala/guvnor-ala-docker-provider/src/main/java/org/guvnor/ala/docker/config/DockerRuntimeConfig.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.guvnor.ala.docker.config;
+
+import org.guvnor.ala.config.ProvisioningConfig;
+import org.guvnor.ala.config.RuntimeConfig;
+
+/*
+ * Represents the Docker Runtime Configuration specific settings to run to Docker Images
+ * @see ProvisioningConfig
+ * @see RuntimeConfig
+ */
+public interface DockerRuntimeConfig extends ProvisioningConfig,
+        RuntimeConfig {
+
+    /*
+     * Get the Docker Image Name to be executed
+     * @return String with the Docker Image name. 
+     */
```

Comment:
Really appreciate the JavaDocs, so just nitpicking. In general, JavaDoc should be written in 3rd person descriptive, not 2nd person e.g. Gets the docker image name to be executed.

For ```@return``` it's always good to indicate if null is possible (the type doesn't need to be part of the JavaDocs as its part of the method signature) e.g. ```@return the docker image name, never null.```

http://www.oracle.com/technetwork/articles/java/index-137868.html